server/src/main/java/org/elasticsearch/search/fetch/FetchPhase.java

```diff
@@ -226,7 +251,7 @@ List<FetchSubPhaseProcessor> getProcessors(SearchShardTarget target, FetchContex
         }
     }
 
-    private static HitContext prepareHitContext(
+    private HitContext prepareHitContext(
```

Comment:
This can stay static or are my eyes playing tricks on me? :)

Reply:
No no, great catch :) 
POC-ing side effect